tox.ini

```diff
@@ -0,0 +1,23 @@
+[tox]
+envlist = py27,py35,flake8
+
+[testenv]
+basedeps = tensorflow==1.0.0
+           protobuf==3.1.0
+           solutionbox/structured_data/
+           solutionbox/image_classification/
+deps = {[testenv]basedeps}
+commands =
+  python ./tests/main.py
+  python ./legacy_tests/main.py
+
+[testenv:py27]
+deps = {[testenv]basedeps}
+       google-cloud-dataflow==0.5.5
+
+[flake8]
```

Comment:
So these were actually the args *to* `flake8` when invoked, but that was a little mysterious.

For now, I just went ahead and inlined those args -- we can get fancier later as needed.